### DISCUSSION TITLE
sort by new does not work on patch overview

### DISCUSSION THREAD

**Discussion by mickeyvanolst** [cables-gl/cables_docs#633] [! Welcome - readme first!]
![image](https://github.com/cables-gl/cables_docs/assets/1071097/6d60435a-aed1-4325-b4ff-97081f112534)

Says latest patch is 8 years ago.

**Comment by pandrr** [comment]
order is by latest change, you are seeing the publishes date. we are currently redoint this stuff.

you can hover the date and see last change/published date etc
